packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -231,11 +260,19 @@ export class VirtualNode extends VirtualNodeBase {
     this.backends.push({
       virtualService: {
         virtualServiceName: virtualService.virtualServiceName,
+        clientPolicy: virtualService.clientPolicy ? {
+          tls: {
+            enforce: virtualService.clientPolicy?.tlsClientPolicy.enforce ?? true,
+            validation: virtualService.clientPolicy?.tlsClientPolicy.validation.bind(this).tlsValidation,
+            ports: virtualService.clientPolicy?.tlsClientPolicy.ports,
+          },
+        } : undefined,
       },
```

Comment:
Exactly the same comments in this class, as in `VirtualGateway`.

In particular, it looks like a fair amount of duplication could be eliminated by moving some of these properties to `bind()`.